src/Middleware/Diagnostics/src/DeveloperExceptionPage/DeveloperExceptionPageMiddlewareImpl.cs

```diff
@@ -101,6 +101,19 @@ public async Task Invoke(HttpContext context)
         }
         catch (Exception ex)
         {
+
+            if (ex is OperationCanceledException && context.RequestAborted.IsCancellationRequested)
+            {
+                _logger.RequestAbortedException();
+
+                if (!context.Response.HasStarted)
+                {
+                    context.Response.StatusCode = 499;
```

Comment:
probably warrants a comment explaining the choice of this specific error-code (is this to mirror nginx client-close?) - or at least to prevent the reader from searching :)

Reply:
Yes , It is same as nginx client-close (client closed request). Sure, I will try to make const Somewhere and put comment

Reply:
Http.StatusCodes

Reply:
```suggestion
                    context.Response.StatusCode = StatusCodes.Status499ClientClosedRequest;
```